packages/x-scheduler/src/joy/internals/components/time-grid/TimeGrid.types.ts

```diff
@@ -0,0 +1,9 @@
+import { SchedulerValidDate } from '../../../../primitives/utils/adapter/types';
```

Comment:
Nitpick: "oh my got" these relative import depths. 🙈 🤯 
It would be interesting to look into a package level alias (something like `~/primitives`) and whether it is properly replaced with the expected relative import at build time. 🤔